packages/react/src/interactions/InlineChoiceInteraction/InlineChoiceInteraction.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import {InlineChoiceInteractionCharacteristics as InlineChoiceInteractionProps} from '@qtikit/model/lib/qti2_2';
+
+const InlineChoiceInteraction: React.FC<InlineChoiceInteractionProps | any> = ({shuffle, required, ...props}) => {
+  const [value, setValue] = React.useState('$choose');
```

Comment:
Wouldn't a placeholder value be null?

Reply:
No, $choose value make the first option item to be a default

Reply:
I know that,
that means How about setting the value of the first option to null?

Reply:
Like this? (value attr doesn't accept null)

```suggestion
  const [value, setValue] = React.useState<string | undefined>(undefined);
  
  return (
    <select value={value} onChange={handleChange}>
      <option>Choose...</option>
      {props.children}
    </select>
  );
```